File: system_details.py
import sys
import sqlite3

sno=sys.argv[1]
sysno=sys.argv[2]
model=sys.argv[3]
m_serial=sys.argv[4]
m_remarks=sys.argv[5]
ram=sys.argv[6]
r_serial=sys.argv[7]
r_remarks=sys.argv[8]
hdd=sys.argv[9]
hddr=sys.argv[10]
hdds=sys.argv[11]
fdd=sys.argv[12]
fdds=sys.argv[13]
fddr=sys.argv[14]
monitor=sys.argv[15]
monitors=sys.argv[16]
monitorr=sys.argv[17]
key=sys.argv[18]
keys=sys.argv[19]
keyr=sys.argv[20]
mouse=sys.argv[21]
mouses=sys.argv[22]
mouser=sys.argv[23]
proce=sys.argv[24]
procer=sys.argv[25]

with sqlite3.connect('lab.db') as db:
    c = db.cursor()

s_found = ('SELECT * FROM sys WHERE sysno = ?')
c.execute(s_found,[(sysno)])
found = c.fetchall()

if found:
    print("a")
else:
    a_f=('INSERT INTO sys(sno,sysno,model,m_serial,m_remark,ram,r_serial,r_remark,hdd,h_serial,h_remark,fdd,f_serial,f_remark,monitor,mon_serial,mon_remark,keyboard,key_serial,key_remark,mouse,mou_serial,mou_remark,process,pro_remark)VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)')
    c.execute(a_f,[(sno),(sysno),(model),(m_serial),(m_remarks),(ram),(r_serial),(r_remarks),(hdd),(hdds),(hddr),(fdd),(fdds),(fddr),(monitor),(monitors),(monitorr),(key),(keys),(keyr),(mouse),(mouses),(mouser),(proce),(procer)])
    
    error = db.commit()

    if not error:
        print("s")     
    else:
        print("n")

sys.stdout.flush()

db.close()

File: system_details.js
function sys(){
    var python = require("python-shell")
    var path=require("path")

    document.getElementById('sys_clear').onclick = function(){
        document.getElementById("s_no").value=""
        document.getElementById("sys_no").value=""
        document.getElementById("model").value=""
        document.getElementById("m_serial").value=""
        document.getElementById("m_remarks").value=""
        document.getElementById("ram").value=""
        document.getElementById("r_serial").value=""
        document.getElementById("r_remarks").value=""
        document.getElementById("hdd").value=""
        document.getElementById("hdd_serial").value=""
        document.getElementById("hdd_remarks").value=""
        document.getElementById("fdd").value=""
        document.getElementById("fdd_serial").value=""
        document.getElementById("fdd_remarks").value=""
        document.getElementById("monitor").value=""
        document.getElementById("monitor_serial").value=""
        document.getElementById("monitor_remarks").value=""
        document.getElementById("keyboard").value=""
        document.getElementById("key_serial").value=""
        document.getElementById("key_remarks").value=""
        document.getElementById("mouse").value=""
        document.getElementById("mouse_serial").value=""
        document.getElementById("mouse_remarks").value=""
        document.getElementById("process").value=""
        document.getElementById("process_remarks").value=""
    }
    document.getElementById('sys_save').onclick = function(){
        
        var sno=document.getElementById("s_no").value
        var sysno=document.getElementById("sys_no").value
        var model=document.getElementById("model").value
        var m_serial=document.getElementById("m_serial").value
        var m_remarks=document.getElementById("m_remarks").value
        var ram=document.getElementById("ram").value
        var r_serial=document.getElementById("r_serial").value
        var r_remarks=document.getElementById("r_remarks").value
        var hdd=document.getElementById("hdd").value
        var hdds=document.getElementById("hdd_serial").value
        var hddr=document.getElementById("hdd_remarks").value
        var fdd=document.getElementById("fdd").value
        var fdds=document.getElementById("fdd_serial").value
        var fddr=document.getElementById("fdd_remarks").value
        var monitor=document.getElementById("monitor").value
        var monitors=document.getElementById("monitor_serial").value
        var monitorr=document.getElementById("monitor_remarks").value
        var key=document.getElementById("keyboard").value
        var keys=document.getElementById("key_serial").value
        var keyr=document.getElementById("key_remarks").value
        var mouse=document.getElementById("mouse").value
        var mouses=document.getElementById("mouse_serial").value
        var mouser=document.getElementById("mouse_remarks").value
        var proce=document.getElementById("process").value
        var procer=document.getElementById("process_remarks").value

        var options = {
            scriptpath : 'E:/Projects/',
            pythonpath : 'C:/Users/LAKSH/AppData/Local/Programs/Python/Python37-32/',
            args : [sno,sysno,model,m_serial,m_remarks,ram,r_serial,r_remarks,hdd,hddr,hdds,fdd,fdds,fddr,monitor,monitors,monitorr,key,keys,keyr,mouse,mouses,mouser,proce,procer]
        } 
     var show=new python.PythonShell("system_details.py", options); 
        show.on('message', function(msg){
            var a=msg
             if(a=="a"){
                alert("already Exist")
            }else if(a=="s"){
                alert("Successfully Created")
            }else{
                alert("Enter the Correct Info")
            }
        }) 
    }

}
function show1(sid){
        var python = require("python-shell")
        var path=require("path")

        var sysno=document.getElementById("sys_no").value
        var options = {
        scriptpath : 'E:/Projects/',
        pythonpath : 'C:/Users/LAKSH/AppData/Local/Programs/Python/Python37-32/',
        args : [sysno]
        } 
    var show=new python.PythonShell("sys_show.py", options); 
        show.on('message', function(msg){
            obj = JSON.parse(msg);
            document.getElementById('id').innerHTML=obj['id'];
            document.getElementById('name').innerHTML=obj['name'];
            document.getElementById('forh').innerHTML=obj['forh'];
            document.getElementById('dob').innerHTML=obj['dob'];
            document.getElementById('age').innerHTML=obj['age'];
            document.getElementById('bg').innerHTML=obj['bg'];
            document.getElementById('mn').innerHTML=obj['mn'];
            document.getElementById('addrs').innerHTML=obj['address'];
            document.getElementById('email').innerHTML=obj['email'];
            document.getElementById('yoe').innerHTML=obj['yoe'];
            document.getElementById('dept').innerHTML=obj['dept'];
            //id,name,forh,dob,age,bg,mn,address,email,yoe,dept
        })
}